src/app/containers/Metadata/index.jsx

```diff
@@ -43,26 +43,25 @@ const MetadataContainer = ({ metadata, promo }) => {
             publishingPrinciples,
             noBylinesPolicy,
           }) => {
-            /* Canonical link generated from servicename and id */
-            const canonicalLink = `https://www.bbc.com/${service}/articles/${id}`;
+            const canonicalLink = `${origin}/${service}/articles/${id}`;
```

Comment:
If this is using `origin` and we expect amp to [use the `default`](https://github.com/bbc/simorgh/pull/1497/files#r270533183) does this not break the expectations?

Reply:
@pjlee11 By default, I mean that `origin` has a default value of `https://www.bbc.co.uk` defined here: https://github.com/bbc/simorgh/blob/af741b24b53c3a9e40385bc50509424437fd3ee0/src/app/contexts/RequestContext/getOriginContext.js#L2

Reply:
Ahhh okay that makes sense, thanks 👍